### PR TITLE
[draft] cuda 124 arm wheel test

### PR DESCRIPTION
**Pull request by tinglvv**
Test https://github.com/pytorch/builder/pull/1775 (merged) for ARM wheel addition
Test https://github.com/pytorch/builder/pull/1804 (WIP) for setting MAX_JOBS

### CODE REVIEW

**Comment by ptrblck** [comment]
I believe the `origin` is set to `https://github.com/pytorch/builder`, which does not contain your branch, so you would need to change it or create a branch in `pytorch/builder` instead of your fork.

File: .github/scripts/generate_binary_build_matrix.py [hunk 1]
```diff
@@ -192,6 +198,7 @@ def translate_desired_cuda(gpu_arch_type: str, gpu_arch_version: str) -> str:
         "cpu-aarch64": "cpu",
         "cpu-cxx11-abi": "cpu-cxx11-abi",
         "cuda": f"cu{gpu_arch_version.replace('.', '')}",
+        "cuda-aarch64": "12.4",
```
Comment (atalman) [inline .github/scripts/generate_binary_build_matrix.py:201]: Same as line above this probably needs to be ``cu124``

**Pull request closed without merging**